docs/site/pages/getting-started.mdx

```diff
@@ -89,112 +139,24 @@ struct MyApp: View {
-dependencies {
-    // Android Player
-    implementation("com.intuit.player.android", "player", androidPlayerVersion)
-    // Optional - reference asset set
-    implementation("com.intuit.player.android", "assets", referenceAssetsVersion)
-}
-```
-
-</Gradle>  
-<Gradle language="Groovy">
-
-```groovy
-// Add Android Player dependencies
-dependencies {
-    // Android Player
-    implementation "com.intuit.player.android:player:$androidPlayerVersion"
-    // Optional - reference asset set
-    implementation "com.intuit.player.android:assets:$referenceAssetsVersion"
-}
-```
-
-</Gradle>
-<Maven>
-
-```xml
-<dependency>
-    <groupId>com.intuit.player.jvm</groupId>
-    <artifactId>core</artifactId>
-    <version>3.2.1</version>
-</dependency>
-```
-
-</Maven>
-</Tabs>
-
-### Configuration
-
-All you need to create an `AndroidPlayer` instance is the Android Context and any plugins you want.
-
-```kotlin
-// create Android Player with reference assets plugin
-val player = AndroidPlayer(context, ReferenceAssetsPlugin())
-```
-
-Once you have a Player, you must add a handler for view updates, which are represented as RenderableAssets. 
-
-```kotlin
-// handle view updates
+// handle view updates - should be done before starting the Player
 player.onUpdate { asset: RenderableAsset? ->
-    // render asset as View
-    val view: View = asset?.render()
-    // add view to screen
+    // render asset as View `into` view tree
+    asset?.render() into binding.playerContainer
 }
-```
-
-### Usage
 
-#### Basic
-With the Android Player and handler configured, all that's left is starting a flow. This is done by calling
-start with a JSON string containing your Player content.
-
-```kotlin
+// start Player
 player.start(content)
 ```
 
-When you're done with Player, release any native runtime memory used to instantiate Player or run the flow.
-
-```kotlin
-player.release()
-```
-
-#### ViewModel
+Creating your own `AndroidPlayer` instance is fairly simple, but it grows in complexity when considering proper resources management and app orchestration. We provide a Fragment/ViewModel integration to make it easier to properly integrate into your app and account for these concerns. Check out the [PlayerViewModel](./player-view-model) section for more information.
+  </android>
+</PlatformTabs>
 
-With the `PlayerViewModel` and `PlayerFragment`, the above orchestration is done for you. All that is needed is to provide concrete implementations of each and add the fragment to your app.
-
-##### PlayerViewModel
-The `PlayerViewModel` is an `AndroidViewModel` meaning it requires the actual Android `Application` as well as an `AsyncFlowIterator` to be supplied in the constructor. The AsyncFlowIterator is what tells Player which flows to run. This can be hardcoded into the view model or expected as an argument, as shown below.
-
-```kotlin
-class SimplePlayerViewModel(application: Application, flows: AsyncFlowIterator) : PlayerViewModel(application, flows) {
```

Comment:
I think this fits configuration for Android. Perhaps add an optional section in configuration for a "quick set-up" or something to include this information

The current section is just player init, which i'd argue isn't very informative at all for how to use

I know this stuff can also fit in render content, so this makes me think we should have a split in each section for `minimal`, `basic`, and `advanced`
`minimal` being `AndroidPlayer(SomePlugin())`, basic being set-up using our provided classes, and advanced can be a brief bit about setting it up yourself from scratch

Since this is just the getting started section, in the advanced section maybe just call out what to look out for if they were to write their own implementation